DependencyInjection/Configuration.php

```diff
@@ -23,6 +23,13 @@ public function getConfigTreeBuilder()
         $rootNode->setBuilder(new AllowExtraPropertiesNodeBuilder());
 
         $rootNode->children()
+            ->arrayNode('allowedParamConverters')
+                ->prototype('array')
+                    ->children()
+                        ->scalarNode('name')->isRequired()->end()
```

Comment:
It does, the json_schema key could be anything, the name is important ;)
So 

```
foo:
    name: foo_param_converter
```

Would be absolutely right. And yes the name is required